Makefile

```diff
@@ -78,6 +78,57 @@ manifests: controller-gen
 generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+# Setup development env
+# Usage: 
+# 	LAPTOP_HOST_IP=<YOUR_LAPTOP_IP> make dev-setup
+# For example:
+#	LAPTOP_HOST_IP=192.168.10.101 make dev-setup
+define TLS_CNF
+[ req ]
+default_bits       = 4096
+distinguished_name = req_distinguished_name
+req_extensions     = req_ext
+[ req_distinguished_name ]
+countryName                = SG
+stateOrProvinceName        = SG
+localityName               = SG
+organizationName           = CAPSULE
+commonName                 = CAPSULE
+[ req_ext ]
+subjectAltName = @alt_names
+[alt_names]
+IP.1   = $(LAPTOP_HOST_IP)
+endef
+export TLS_CNF
+dev-setup:
```

Comment:
The reason why I don't add the dependency to `deploy` is that occasionally I would run into errors while executing `make deploy` because of race condition:

```log
...
deployment.apps/capsule-controller-manager created
mutatingwebhookconfiguration.admissionregistration.k8s.io/capsule-mutating-webhook-configuration created
validatingwebhookconfiguration.admissionregistration.k8s.io/capsule-validating-webhook-configuration created
error: unable to recognize "config/install.yaml": no matches for kind "CapsuleConfiguration" in version "capsule.clastix.io/v1alpha1"
make: *** [deploy] Error 1
```

So in the doc, I mention a retry of `deploy` might be required before executing `dev-setup`

Reply:
Yes, it's something we've already faced in another issue (#368). Nothing so critical, IMO.